helper_test.go

```diff
@@ -0,0 +1,21 @@
+package gotwilio
+
+import (
+	"os"
+	"testing"
+)
+
+var (
```

Comment:
@sfreiberg Going to have to reconcile this with changes I made on the Phone Number Management branch. I wanted to keep them separate, but they share some helper code in the tests, so there will likely be some conflicts (not merge conflicts, since they're in different files, but Go compiler conflicts).

Reply:
Ok, I just merged the other so feel free to rebase against master.